docker/sawtooth-debug-python

```diff
@@ -70,7 +68,6 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
     python3-zmq-dbg \
     python3-cffi-backend-dbg python3-apt-dbg \
     python3-netifaces-dbg \
-    libzmq-dbg python3-cchardet-dbg \
```

Comment:
libzmq-dbg should stay?

Reply:
This PR is solely to remove chardet.  I recommend we investigate other opportunities to remove unused components separately.

Reply:
Confirming that only chardet is to be removed.  cchardet should not be removed, as it is a dependency of aiohttp.